### PR TITLE
Add trivia winners, sponsors and organizers.

### PR DESCRIPTION
**Pull request by TraceyHolinka**
Reorganized this a bit.

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -2,3 +2,13 @@
 	from = "/discord"
 	to = "https://discord.gg/6MZWP8z"
 	status = 301
+
```
Comment (jackkoppa) [inline netlify.toml:5]: @TraceyHolinka, per the build log, I think we actually need to remove these newlines? https://app.netlify.com/sites/vuedc/deploys/5f66639b08ea9e0007a75a60 

> 4:01:42 PM: Invalid netlify.toml file: Error while decoding file /opt/build/repo/netlify.toml: Near line 7 (last key parsed 'redirects.from'): strings cannot contain newlines

**Review by jackkoppa — COMMENTED** [review]
Awesome, thanks @TraceyHolinka! We might just need to get the build working by removing newlines

**Update:** that part is fixed, but I think we need Netlify to use a newer version of yarn; that might be on Ben currently 🤔

**Comment by TraceyHolinka** [comment]
> Awesome, thanks @TraceyHolinka! We might just need to get the build working by removing newlines

Thanks @jackkoppa ! I have no experience with netlify.

**Review by bencodezen — APPROVED** [review]
Thanks for doing this @TraceyHolinka! Looks great!

**Comment by bencodezen** [comment]
@TraceyHolinka It looks like the deploy preview failed. Did you have any issues locally?

**Comment by TraceyHolinka** [comment]
> @TraceyHolinka It looks like the deploy preview failed. Did you have any issues locally?

I didn't have any issue locally. @jackkoppa  thought it was differences in versions of yarn and something needed to be done on the Netlify side. Here's the error message:

"Error during Yarn install"

**Pull request merged**